game/scripts/npc/abilities/nevermore_requiem.txt

```diff
@@ -23,7 +23,7 @@
 
     // Damage.
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityDamage"                                       "80 160 240 320 480" //OAA
+    "AbilityDamage"                                       "80 160 240 320 400" //OAA, applies magic resist reduction
```

Comment:
was an overnerf from me to set max level this low imo. really falls off into oaa levels (pipe), don't think it needs to be nerfed again and maybe needs a buff at max

Reply:
This is dmg per soul and balancing it is tricky because magic resist reduction is involved. Maybe you nerfed the wrong thing, maybe the dmg is too high at early lvls and not OAA lvls? Maybe 80/150/220/360/500.